classes/xslt/XMLTypeDescription.inc.php

```diff
@@ -98,14 +141,18 @@ function parseTypeName($typeName) {
 	}
 
 	/**
-	 * @copydoc TypeDescription::checkType()
+	 * Validates an object against the internal type description.
+	 *
+	 * @param $object mixed
+	 * @param $noValidation boolean If set to true no XML validation will be done
+	 * @return boolean
 	 */
-	function checkType(&$object) {
+	function checkType(&$object, $noValidation = null) {
 		// We only accept DOMDocument objects and source strings.
 		if (!is_a($object, 'DOMDocument') && !is_string($object)) return false;
 
 		// No validation...
-		if ($this->_validationStrategy == XML_TYPE_DESCRIPTION_VALIDATE_NONE) return true;
+		if ($this->_validationStrategy == XML_TYPE_DESCRIPTION_VALIDATE_NONE || $noValidation) return true;
```

Comment:
@asmecher, it would be possible to set the validationStrategy to `XML_TYPE_DESCRIPTION_VALIDATE_NONE` before calling the `isCompatible` function in the `NativeExportFilter`. This would reduce all the changes in pkp-lib -- only `setValidationStrategy` would be needed -- thus I suppose it is better to do it that way, right? I will then change it this way, OK?